packages/block-editor/src/components/inserter/style.scss

```diff
@@ -268,15 +268,11 @@ $block-inserter-tabs-height: 44px;
 	}
 }
 
-.block-editor-inserter__block-patterns-tabs-container,
-.block-editor-block-patterns-explorer__sidebar {
+.block-editor-inserter__block-patterns-tabs-container {
 	height: 100%;
 	nav {
 		height: 100%;
 	}
-	.block-editor-block-patterns__source-filter select.components-select-control__input {
-		height: 40px;
-	}
```

Comment:
I searched the whole project for the keyword `source-filter`, but there doesn't seem to be any element with a class name containing that keyword. It [existed in the past](https://github.com/WordPress/gutenberg/pull/53835/files#diff-904624e5448ae785a390874f8e080f5eb5c6fca7378cfbeb64392ba17a692f6cR31), but I don't think it exists now.